scripts/flink-image-install.sh

```diff
@@ -8,4 +8,19 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 set -ex
 ROOT_DIR=$(readlink -f $(dirname $0)/..)
-kubectl apply -f ${ROOT_DIR}/flink-image/ClusterFlinkImage.yaml
+source ${ROOT_DIR}/scripts/env.sh
+# Set the docker registry with desa installe
+echo sdp install script $SDP_INSTALL_SCRIPT
+$SDP_INSTALL_SCRIPT config set registry ${DOCKER_REGISTRY}
+
+# Push Image to docker registry with desa installer
+$SDP_INSTALL_SCRIPT push --input $DOCKER_IMAGE_TAR  --ca-certs-dir $CERTS_PATH
+
+if [ $? -ne 0 ]
```

Comment:
There is no need for checking the return value and using an if statement. `set -ex` at the top will immediately stop and if any errors occur.

Reply:
removed set -ex as per previous comment. Let us keep this if statement because sometimes push may fail.

Reply:
If the push fails and returns non-zero, the script will stop. It will never execute the if statement.